fe1-web/src/core/objects/Server.ts

```diff
@@ -0,0 +1,63 @@
+import { RemoveMethods } from 'core/types';
+
+import { PublicKey } from './PublicKey';
+
+export type ServerAddress = string;
+
+export interface ServerState {
+  address: string;
+  publicKey: string;
+}
+
+export class Server {
+  /**
+   * The canonical address of the server
+   */
+  address: ServerAddress;
```

Comment:
should this be a `URL` object ?

Reply:
I guess that could work..? 😅 I will have to check the URL type since I am not sure what it does and whether it is supported on mobile devices